lib/structures/Member.js

```diff
@@ -41,6 +41,9 @@ const VoiceState = require("./VoiceState");
 class Member extends Base {
     constructor(data, guild, client) {
         super(data.id || data.user.id);
+        if(!data.id && data.user) {
+            data.id = data.user.id;
```

Comment:
I'm sorry to comment only now a merged PR, but couldn't that have been solved by simply using `this.id` in the constructor instead?